pandas/tests/frame/test_reductions.py

```diff
@@ -285,16 +285,11 @@ def test_stat_op_api(self, float_frame, float_string_frame):
         assert_stat_op_api("sem", float_frame, float_string_frame)
         assert_stat_op_api("median", float_frame, float_string_frame)
 
-        try:
-            from scipy.stats import (  # noqa:F401
-                kurtosis,
-                skew,
-            )
-
-            assert_stat_op_api("skew", float_frame, float_string_frame)
-            assert_stat_op_api("kurt", float_frame, float_string_frame)
-        except ImportError:
-            pass
+    @pytest.mark.filterwarnings("ignore:Dropping of nuisance:FutureWarning")
```

Comment:
why is this necessary?

Reply:
I'm getting
```
pandas/tests/frame/test_reductions.py::TestDataFrameAnalytics::test_stat_op_api_skew_kurt
  /pandas/tests/frame/test_reductions.py:160: FutureWarning: Dropping of nuisance columns in DataFrame reductions (with 'numeric_only=None') is deprecated; in a future version this will raise TypeError.  Select only valid columns before calling the reduction.
    getattr(float_string_frame, opname)(axis=0)
```

without it. Copying it from the original larger test, guessing it's based on the fixture data and `assert_stat_op_api`

Reply:
Ah, I hadn't noticed it was part of the original test, thanks for explaining